### ISSUE TITLE
Display absolute path even when relative_path = TRUE

### ISSUE THREAD

**Issue by renkun-ken** [jimhester/covr#84]
```
> library(covr)
> package_coverage()
Test Coverage: 95.97%
C:\Users\Kun\Documents\Workspaces\formattable\R\utils.R: 88.89%
C:\Users\Kun\Documents\Workspaces\formattable\R\formattable.R: 93.33%
C:\Users\Kun\Documents\Workspaces\formattable\R\formats.R: 100.00%
C:\Users\Kun\Documents\Workspaces\formattable\R\formatter.R: 100.00%
C:\Users\Kun\Documents\Workspaces\formattable\R\style.R: 100.00%
C:\Users\Kun\Documents\Workspaces\formattable\R\transform.R: 100.00%
> package_coverage(relative_path = TRUE)
Test Coverage: 95.97%
C:\Users\Kun\Documents\Workspaces\formattable\R\utils.R: 88.89%
C:\Users\Kun\Documents\Workspaces\formattable\R\formattable.R: 93.33%
C:\Users\Kun\Documents\Workspaces\formattable\R\formats.R: 100.00%
C:\Users\Kun\Documents\Workspaces\formattable\R\formatter.R: 100.00%
C:\Users\Kun\Documents\Workspaces\formattable\R\style.R: 100.00%
C:\Users\Kun\Documents\Workspaces\formattable\R\transform.R: 100.00%
```

It looks like `relative_path = TRUE` does not work? `zero_coverage()` also shows the absolute paths with the coverage results.

**Comment by jimhester** [comment]
I am guessing it is an issue with Windows file separators being backslashes
rather than forward slashes.  Relative paths work fine on Linux and OSX.  I
will look into it soon.

**Comment by renkun-ken** [comment]
Thanks! I guess `normalizePath()` may work in this case.

**Comment by jimhester** [comment]
This should be fixed with 29b289f9827b151138e41ee3ded4b409945cce92

**Comment by renkun-ken** [comment]
Thanks!
